### PR TITLE
[RFC] Enable easy mirror host customization

### PR DESCRIPTION
**Pull request by vincent-legoll**

### CODE REVIEW

**Review by racciari — COMMENTED** [review]
This should be in the defaults/main.yml to be modified.

File: vars/main.yml [hunk 1]
```diff
@@ -5,7 +5,10 @@
 # See tasks/{RedHat,Debian,Ubuntu}.yml
 openio_repository_creds: "{% if repo.value.user is defined and repo.value.password is defined %}{{ repo.value.user }}:{{ repo.value.password }}@{% endif %}"
 
-openio_repository_mirror_url_base: "http://{{ openio_repository_creds }}mirror.openio.io/pub/repo/openio"
-openio_repository_mirror_url_base_nocreds: "http://mirror.openio.io/pub/repo/openio"
+openio_repository_mirror_host: "mirror.openio.io"
```
Comment (racciari) [inline vars/main.yml]: This should be in the defaults/main.yml

File: vars/main.yml [hunk 2]
```diff
@@ -5,7 +5,10 @@
...
+openio_repository_mirror_url_base: "http://{{ openio_repository_creds }}{{ openio_repository_mirror_host }}/pub/repo/openio"
```
Comment (racciari) [inline vars/main.yml]: I suppose we can change the path `/pub/repo/openio` to a parameter aswell.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ An Ansible role for setup OpenIO's repositories (and OpenStack). Specifically, t
 | `openio_repository_manage_openstack_repository` | `true` | Allow the installation of OpenStack's repository |
 | `openio_repository_openstack_release` | `pike` | OpenStack release version codename |
 | `openio_repository_products` | `{'sds': {'release': '17.04'}}` | OpenIO products repositories to setup |
+| `openio_repository_mirror_host` | `"mirror.openio.io"` | The mirror host to setup (alternative: `"mirror2.openio.io"`) |
```
Comment (racciari) [inline README.md]: Please don't show mirror2 as an alternative ;-)

**Review by racciari — CHANGES_REQUESTED** [review]

File: vars/main.yml (hunk 2 above)
Comment (vincent-legoll) [reply vars/main.yml]: Yes, I agree, we will need this someday, but in another commit/PR, no need to mix things up

**Pull request merged**